polling_stations/apps/data_collection/management/commands/import_knowsley.py

```diff
@@ -3,7 +3,40 @@
 
 class Command(BaseXpressDemocracyClubCsvImporter):
     council_id = "E08000011"
-    addresses_name = "local.2018-05-03/Version 1/Democracy_Club__03May2018 Knowsley.CSV"
-    stations_name = "local.2018-05-03/Version 1/Democracy_Club__03May2018 Knowsley.CSV"
-    elections = ["local.2018-05-03"]
+    addresses_name = "local.2019-05-02/Version 1/Democracy_Club__02May2019Knowsley.CSV"
+    stations_name = "local.2019-05-02/Version 1/Democracy_Club__02May2019Knowsley.CSV"
+    elections = ["local.2019-05-02"]
     csv_delimiter = ","
+    private = True
```

Comment:
This needs removing as we want to get the data from S3 by default.
I wonder if there is something cunning we can do like move this to a global setting you can set in `local.py` or add a lint rule to fail the build if `private = True` is set in an import script. I'll have a quick think tomorrow

Reply:
I'll bung something in a pre-push hook to keep me right for now - was a classic "I'm already late for where I'm meant to be PR..."